randovania/games/planets_zebeth/gui/dialog/cosmetic_patches_dialog.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from randovania.games.planets_zebeth.layout.planets_zebeth_cosmetic_patches import PlanetsZebethCosmeticPatches
+from randovania.gui.dialog.base_cosmetic_patches_dialog import BaseCosmeticPatchesDialog
+from randovania.gui.generated.planets_zebeth_cosmetic_patches_dialog_ui import Ui_PlanetsZebethCosmeticPatchesDialog
+
+if TYPE_CHECKING:
+    from PySide6.QtWidgets import QWidget
+
+    from randovania.layout.base.cosmetic_patches import BaseCosmeticPatches
+
+
+class PlanetsZebethCosmeticPatchesDialog(BaseCosmeticPatchesDialog, Ui_PlanetsZebethCosmeticPatchesDialog):
+    _cosmetic_patches: PlanetsZebethCosmeticPatches
+
+    def __init__(self, parent: QWidget, current: BaseCosmeticPatches):
+        super().__init__(parent)
+        self.setupUi(self)
+
+        assert isinstance(current, PlanetsZebethCosmeticPatches)
+        self._cosmetic_patches = current
+
```

Comment:
As the selector is already in the GUI, I suggest to also fill it and to not have an empty selector.
```suggestion
        for room_gui_type in PlanetsZebethRoomGuiType:
            self.room_name_dropdown.addItem(room_gui_type.long_name, room_gui_type)
```